### ISSUE TITLE
[form-core] add fieldset label as suffix

### ISSUE THREAD

**Issue by tlouisse** [ing-bank/lion#1162]
## Expected behavior

Just like with native fieldset legends, the group label should be pronounced by the screen reader after the label of all children.
Almost similar to https://www.w3.org/WAI/tutorials/forms/grouping/, but as a suffix instead of a prefix (so that the unique part is read first, just like in a native fieldset)

## Actual Behavior

Only the label of the child is read.
Children of of a LionFieldset (implemented with role=group) should add a id reference to each of their parent labels, so the screen reader will read it like a suffix. 
Most likely it could be done via a construction like (following code should run inside a child of a group): `this.addToAriaLabelledBy(this.__parentFormGroup._labelNode)`
